### ISSUE TITLE
[REQUEST] How to structure a large Express app

### ISSUE THREAD

**Issue by kensnyder** [UtahJS/meetups-slc-group#10]
Express isn't very opinionated and leaves many without good direction about how to keep your routes and handlers organized in a large app.

**Comment by iammerrick** [comment]
Also "Express isn't very opinionated" try hapijs :-)

**Comment by wdjungst** [comment]
Agreed.  We could try any wrapper there are so many.  I think the question is what type of app?  Is this an MVC app?  Is this a Redux app?  I would structure those very differently.

**Comment by iammerrick** [comment]
@wdjungst  but hapi isn't a wrapper, it's a different tool/paradigm :-)

**Comment by iammerrick** [comment]
I don't see value in express/koa when I can leverage hapijs 👍
